lib/facter/pe_status_check.rb

```diff
@@ -219,6 +219,19 @@
     end
   end
 
+  chunk(:S0029) do
+    next unless PEStatusCheck.replica? || PEStatusCheck.postgres? || PEStatusCheck.primary?
+    # check if concurrnet connections to Postgres approaching 90% defined
+
+    maximum = PEStatusCheck.max_connections.to_i
+    current = PEStatusCheck.cur_connections.to_i
+    percent_used = (current / maximum) * 100
+    { S0029: percent_used >= 90 }
```

Comment:
Iv tested this in my lab:
maximum = 400
current  = 40
percent_used = 0

the first two functions appear to work and get figures, but percent_used should be 10, and this expression is giving 0 so we have an issue here.

Its because the data type you have set is integer, so when the math brings up 0.1 it rounds to 0